### PR TITLE
Dependencies update and add support  for HTML

### PR DESCRIPTION
**Pull request by matteobaccan**
Hi

I have modified the plugin to support HTML and for update some external library

all the best
matteo baccan

### CODE REVIEW

**Comment by matteobaccan** [comment]
I have fixed the list of a test result that is changed with new Java core

File: src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml [hunk 1]
```diff
@@ -20,6 +20,10 @@
                 <file name="com-github-philippefichet-sonarlint4netbeans-SonarLintParserResultTaskFactory.instance">
                 </file>
             </folder>
+            <folder name="x-html">
```
Comment (philippefichet) [inline src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml:23]: From https://developer.mozilla.org/fr/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types, text/x-html does not exist and not work with netbeans annotation on a .html file.
On Ubuntu, mimetype for html are:
```
find /usr/share/mime -name "*html*"
/usr/share/mime/application/xhtml+xml.xml
/usr/share/mime/application/vnd.ms-htmlhelp.xml
/usr/share/mime/text/html.xml
/usr/share/mime/text/htmlh.xml
```
Wouldn't it be more just "html"?

Comment (matteobaccan) [reply src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml]: Hi Philippe

maybe. I have tested the update on Windows 10 and I have not seen any difference.
The correct mime is HTML, so I have fixed the XML

Can you check if on Ubuntu now works fine?

Excuse me for the mistake

all the best
matteo baccan

Comment (philippefichet) [reply src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml]: The typemime is used here to launch an analysis and display the results as annotations (icon on the left in the editor).
It works perfectly under Ubuntu ;)
Thanks for your pull request :) 👍

Comment (matteobaccan) [reply src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml]: Very good :)

I have also try to add some other analyzers without success, I'll retry in future

ciao
matteo

**Comment by philippefichet** [comment]
Thanks again :)

File: src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml (hunk 1 above)
Comment (philippefichet) [reply src/main/resources/com/github/philippefichet/sonarlint4netbeans/layer.xml]: Indeed, it is a limitation of this class https://github.com/SonarSource/sonarlint-core/blob/master/core/src/main/java/org/sonarsource/sonarlint/core/client/api/common/Language.java
All plugins can't be added :(

**Pull request merged**